index.js

```diff
@@ -64,6 +64,7 @@ module.exports = function configParser(yaml, templateFactory) {
             const res = {
                 annotations: Hoek.reach(doc, 'annotations', { default: {} }),
                 jobs: Hoek.reach(doc, 'jobs'),
+                scmUrls: Hoek.reach(doc, 'scmUrls', { default: [] }),
```

Comment:
Can we make this optional? if `scmUrls` not exist, don't bother to add it to res at all.